backend/models/time.py

```diff
@@ -67,10 +67,10 @@ def _get_matrix_http(
         url=request_url.format(points=coordinates),
         params=params
     )
-
     try:
         response.raise_for_status()
-    except requests.HTTPError:
+    except requests.HTTPError as error:
+        logger.warning('Measure API request error - {}'.format(error))
```

Comment:
why warning and not error?

Reply:
Agreed that this is a little sloppy (as Eric is aware) -- the retry is triggered by returning `None`, so the error is squelched and `None` is returned. 

Perhaps add a FIXME for a more meaningful retry condition?

Reply:
should it be at least a logging.error though? doesn't have to be an exception

Reply:
Okay, both `logger.error` and `FIXME` then. ;)